packages/dd-trace/test/appsec/blocking.spec.js

```diff
@@ -277,3 +248,41 @@ describe('blocking', () => {
     })
   })
 })
+
+describe('waf actions', () => {
```

Comment:
you're not testing that block has priority over redirect right ?

Reply:
WAF resolves blocking action priority internally, so no need to test.

> This means the caller no longer has to translate action IDs to their relevant definition and any blocking action conflicts are resolved internally following a simple set of rules:
> - When multiple actions of the same type are present, the first one produced has precedence.
> - An action of type redirect_request has priority over an action of type block_request.

[Upgrading guide](https://github.com/DataDog/libddwaf/blob/1.17.0/UPGRADING.md#upgrading-from-116x-to-1170)

Reply:
Oh you're right my bad, it's this code that made me think you were prioritizing the block request:
![image](https://github.com/DataDog/dd-trace-js/assets/70517128/640aab10-a613-4805-a993-6eec70de2738)
but in reality you're just trying to find any of the two, the order doesn't matter